### ISSUE TITLE
Support regionprops properties in region adjacency graphs (RAGs)

### ISSUE THREAD

**Issue by WMF1997** [scikit-image/scikit-image#4364]
hello, 
I want to get the centroid of all the superpixels so that I can convert an image to a graph.

And I use ```skimage.segmentation.slic``` to segment the image, then ```skimage.future.graph.rag.rag_mean_color``` to get the mean color of the superpixels. but i want the node coordinate, edge weight and weight connection. And I use ```skimage.future.graph.show_rag(labels, rag, image)``` to get the ```'centroid'``` key of the graph. But the graph have to be shown due to ```import matplotlib.pyplot as plt```. 

I know that, 
first, it should be a ```networkx``` issue rather than a ```skimage``` issue, 
second, implementing this is very easy: just comment the ```matplotlib``` part is okay. 

And I what I want to request is that ```skimage.future.graph.rag.noshow_rag```, which does not show the final visualization result, only for getting ```centroid``` key and something else. Or, adding a paramter```show``` in ```skiamge.future.graph.show_rag```, and the default of ```show``` is ```True```. 

Yours, 
@wmf1997

**Comment by jni** [comment]
@WMF1997 I don't think this is a networkx issue at all, indeed this should be reasonably easy to compute within scikit-image. Thanks for the report!

Actually I think what you are after should not have the word `show` in it at all. Computing centroids is not necessarily about showing!

As an early workaround, you can actually compute the centroids yourself quite easily as follows:

```python
from skimage import measure

properties = measure.regionprops(label_image)

for prop in properties:
    rag.nodes[prop.label]['centroid'] = prop.centroid
```

So, depending on what properties you need, I think actually your issue is possibly best summarised as: "add region properties to RAG". The snippet above could be generalised to every property supported by regionprops, which actually would be quite nice!

What do you think?

**Comment by WMF1997** [comment]
Hello @jni 
Thank you for telling me the way of calculating every property supported by regionprops in ```skimage.measure.regionprops```

**Comment by jni** [comment]
@WMF1997 I'm going to reopen this, as I think it would be nice to have this built into RAG. But I'm happy that my code snippet helps you! I hope we can make this even more seamless for future users.
